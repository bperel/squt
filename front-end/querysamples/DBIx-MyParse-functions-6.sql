SELECT DATE_ADD('1998-01-02', INTERVAL 31 DAY)
