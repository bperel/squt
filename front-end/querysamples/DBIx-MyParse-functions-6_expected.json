{
  "warning": {
    "No alias": {
      "date_add_interval": ["function", "SELECT"]
    }
  },
  "functions": {
    "0": {
      "to": "output",
      "constants": {
        "interval_day": "interval_day",
        "31": "31",
        "1998-01-02": "1998-01-02"
      },
      "name": "date_add_interval",
      "alias": "0"
    }
  }
}